src/test/kotlin/it/pagopa/ecommerce/eventdispatcher/services/NodeServiceTests.kt

```diff
@@ -63,24 +63,48 @@ class NodeServiceTests {
             TRANSACTION_ID))
         .willReturn(events.toFlux())
 
-      given(nodeClient.closePayment(any())).willReturn(Mono.just(closePaymentResponse))
+      given(nodeClient.closePayment(capture(closePaymentRequestCaptor)))
+        .willReturn(Mono.just(closePaymentResponse))
 
       /* test */
       assertEquals(
         closePaymentResponse,
         nodeService.closePayment(TransactionId(transactionId), transactionOutcome))
+
+      assertEquals(transactionId, closePaymentRequestCaptor.value.transactionId)
+      assertEquals(OutcomeEnum.KO, closePaymentRequestCaptor.value.outcome)
+      // check additionalPaymentInformations
+      assertNull(closePaymentRequestCaptor.value.additionalPaymentInformations)
+      // check transactionDetails
+      assertEquals(
+        UserDto.TypeEnum.GUEST, closePaymentRequestCaptor.value.transactionDetails.user.type)
+      assertEquals(
+        TransactionDetailsStatusEnum.TRANSACTION_DETAILS_STATUS_CANCELED.status,
+        closePaymentRequestCaptor.value.transactionDetails.transaction.transactionStatus)
+      assertEquals(
+        Transaction.ClientId.CHECKOUT.name,
+        closePaymentRequestCaptor.value.transactionDetails.info.clientId)
+      assertEquals(TIPO_VERSAMENTO_CP, closePaymentRequestCaptor.value.transactionDetails.info.type)
+      assertEquals(
+        closePaymentRequestCaptor.value.transactionDetails.transaction.amount,
+        closePaymentRequestCaptor.value.transactionDetails.transaction.grandTotal)
+      assertNull(closePaymentRequestCaptor.value.transactionDetails.transaction.fee)
+      assertNotNull(closePaymentRequestCaptor.value.transactionDetails.transaction.amount)
+      assertNotNull(closePaymentRequestCaptor.value.transactionDetails.transaction.grandTotal)
```

Comment:
```suggestion
assertNull(closePaymentRequestCaptor.value.transactionDetails.transaction.fee)
      val paymentNoticesAmount = EuroUtils.euroCentsToEuro(activatedEvent.data.paymentNotices.stream().mapToInt{it.amount}.sum())
      assertEquals(paymentNoticesAmount,closePaymentRequestCaptor.value.transactionDetails.transaction.amount)
      assertEquals(paymentNoticesAmount,closePaymentRequestCaptor.value.transactionDetails.transaction.grandTotal)
```
Perform check that both amount and grandTotal are equals to payment notices amount sum

Reply:
Fixed with 👉🏻 2610b8c15e08d4983d5171bf6bda99f7251f1956